scala/src/main/javadocstub/JavaDocStub.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
// comments result in warning. @tawan0109  fixed those issues in https://github.com/Microsoft/SparkCLR/commit/721a63b74eddbf1a2bd1fbd877c74ce49595eaef using /* */ style commenting.